tests/ert/unit_tests/gui/tools/plot/test_plot_api.py

```diff
@@ -275,6 +293,75 @@ def test_plot_api_big_summary_memory_usage(
     assert total_memory_usage < max_memory_mb
 
 
+def test_plotter_on_all_snake_oil_responses_time(api_and_snake_oil_storage):
+    api, _ = api_and_snake_oil_storage
+    t0 = time.time()
+    key_infos = api.all_data_type_keys()
+    all_ensembles = api.get_all_ensembles()
+    t1 = time.time()
+    # Cycle through all ensembles and get all responses
+    for key_info in key_infos:
+        for ensemble in all_ensembles:
+            api.data_for_key(ensemble_id=ensemble.id, key=key_info.key)
+
+        if key_info.observations:
+            with contextlib.suppress(RequestError, TimeoutError):
+                api.observations_for_key(
+                    [ens.id for ens in all_ensembles], key_info.key
+                )
+
+        # Note: Does not test for fields
+        if not (str(key_info.key).endswith("H") or "H:" in str(key_info.key)):
+            with contextlib.suppress(RequestError, TimeoutError):
+                api.history_data(
+                    key_info.key,
+                    [e.id for e in all_ensembles],
+                )
+
+    t2 = time.time()
+    time_to_get_metadata = t1 - t0
+    time_to_cycle_through_responses = t2 - t1
+
+    assert time_to_get_metadata < 1  # 0.09 on py312 macbook pro m1 max
+    assert time_to_cycle_through_responses < 14  # 4.48 on py312 macbook pro m1 max
+
+    gc.collect()
```

Comment:
Do you need this? You already call it in the fixture

Reply:
Oops, removed it!